src/devsecops/base/base_handler.py

```diff
@@ -8,6 +8,7 @@
 import logging.handlers
 import sys
 import urllib3
+import os
```

Comment:
Can you change this to `os.path`? Not all versions of python bring `path` into the `os` namespace by default, so it's nice to be as specific as we have to here.